src/NSwag.AspNetCore/SwaggerUiSettingsBase.cs

```diff
@@ -35,9 +35,7 @@ public abstract class SwaggerUiSettingsBase : SwaggerSettings
         public SwaggerUiSettingsBase()
         {
             TransformToExternalPath = (internalUiRoute, request) =>
-            {
-                return request.GetBasePath() + internalUiRoute;
-            };
+                request.GetBasePath().TrimEnd('/') + '/' + internalUiRoute.TrimStart('/');
```

Comment:
Do we need null checks?

Reply:
Good idea. I'll add that.

Reply:
I'm usually defensive here... checking the docs/source code might be another option :-)

Reply:
Yeah, agree with the defensive approach. I've added the null handling, so if both sides are null then this will output `"/"`. I don't see a situation where the internal route would be `null` and I'm pretty sure `GetBasePath` at least returns empty string or "/" anyway depending on which case it falls into.